server/gui/main.py

```diff
@@ -237,13 +286,208 @@ def onError(self, err, server_error=False):
             self.serverError = True
             # Report error and switch to load screen
             self.window().shutdownServer()
+        self.resetProgress()
         self.window().stacked_layout.setCurrentIndex(LOAD_INDEX)
         self.error_label.setText(f"Error: {err}")
         self.error_label.resize(self.MAX_CONTENT_WIDTH, self.error_label.height())
+        # self.error_label.setFrameStyle(QFrame.Panel)
 
     onServerError = partialmethod(onError, server_error=True)
 
 
+class AdvancedOptionsArea(QFrame):
```

Comment:
As an alternative to this relatively complex property sheet, have you considered a simple text box that just accepts the CLI text parameters?    Advanced options are "advanced", and so there is less of a usability concern.  And it has the advantage of making all CLI params, now and future, available for use without a Qt implementation.

Reply:
Love that idea! Will re-implement